mesh/meshdb.go

```diff
@@ -403,18 +403,26 @@ func (m *MeshDB) writeTransactions(l types.LayerID, txs []*types.Transaction) er
 	return nil
 }
 
-func (m *MeshDB) writeTransactionRewards(l types.LayerID, accounts []types.Address, reward *big.Int) error {
+type dbReward struct {
+	TotalReward         uint64
```

Comment:
add a comment about the meaning of these fields as I'm a bit confused. layerReward - totalRewads = tx fees?